packages/venia-concept/src/index.js

```diff
@@ -45,14 +45,22 @@ const apolloLink = ApolloLink.from([
     Adapter.apolloLink(apiBase)
 ]);
 
-ReactDOM.render(
-    <Adapter apiBase={apiBase} apollo={{ link: apolloLink }} store={store}>
-        <AppContextProvider>
-            <App />
-        </AppContextProvider>
-    </Adapter>,
-    document.getElementById('root')
-);
+const renderApp = Component =>
+    ReactDOM.render(
+        <Adapter apiBase={apiBase} apollo={{ link: apolloLink }} store={store}>
+            <AppContextProvider>
+                <Component />
+            </AppContextProvider>
+        </Adapter>,
+        document.getElementById('root')
+    );
+
+renderApp(App);
+if (module.hot) {
+    module.hot.accept('@magento/venia-ui/lib/components/App/container', () =>
```

Comment:
I need to double check that this is the correct usage.

Reply:
`module.hot.accept()` may also work but I'm concerned about the other side effects such as complete reloading of the graphql store.

Reply:
Turns out this is fine, and even necessary, if we want changes to store to trigger hot reload.